package.json

```
@@ -5,14 +5,17 @@
   "dependencies": {
     "@material-ui/core": "^4.6.0",
     "@material-ui/icons": "^4.5.1",
+    "@reduxjs/toolkit": "^1.0.4",
     "aws-amplify": "^1.1.40",
     "react": "^16.9.0",
     "react-bootstrap": "^0.32.4",
     "react-dom": "^16.9.0",
+    "react-redux": "^7.1.3",
```

```suggestion
    "react-redux": "7.1.3",
```
smaller libraries or dev libraries might be okay but big impactful ones, better to play safe and be more predictable.

---

Done :smile: